style.css

```diff
@@ -1,39 +1,124 @@
-@import url('https://fonts.googleapis.com/css2?family=Inter');
-body{
-    box-sizing: border-box;
+@import url("https://fonts.googleapis.com/css2?family=cyrillic Roboto");
+@import url("https://fonts.googleapis.com/css2?family=Poppins");
+@import url("https://fonts.googleapis.com/css2?family=Crete Round");
+
+body {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  background-repeat: no-repeat;
+}
+
+.container {
+  border: 1px white;
+  width: 100vw;
+  height: 640px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.header {
+  background: #3c3a39;
+  height: 50px;
+  width: 375px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.header-p {
+  margin-left: 15px;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 24px;
+  color: #fff;
 }
-ul li{
-    list-style-type: none;
-    margin-right: 15px;
+
+.fa-bars-icon {
+  margin-right: 15px;
+  font-size: 18px;
+  color: #fff;
 }
-a{
-    text-decoration: none;
-    font-family: Inter;
-    color: white;
+
+.section-1 {
+  background-size: 406px 311px;
+  background-color: black;
+  height: 590px;
+  width: 375px;
 }
-main{
-    background: #1C1A19;
 
+.headline {
+  display: flex;
+  flex-direction: column;
+  align-content: center;
+  justify-content: center;
+  height: 590px;
+  width: 375px;
 }
-header{
-    background: #3E3C3C;
-    color: white;
-    display: flex;
-    padding-left: 30px;
+
+.Primary-text {
+  width: 327px;
+  height: 124px;
+  font-family: "Crete Round", cursive;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 56px;
+  line-height: 64px;
+  letter-spacing: 0.37px;
+  color: #ff6b00;
+  margin-left: 20px;
+  margin-bottom: 10px;
 }
-.header-nav-div{
-    margin-left: 30%;
+
+.Supporting-text {
+  width: 327px;
+  height: 27px;
+  margin-left: 20px;
+  margin-bottom: 10px;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 24px;
+  line-height: 32px;
+  color: #fff5e1;
 }
-.header-name-div p{
-    font-family: Inter;
-    color: white;
+
+.Description {
+  width: 327px;
+  height: 123px;
+  margin-left: 20px;
+  margin-bottom: 10px;
+  font-family: "Poppins", serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #fff;
+}
+
+.social-media {
+  border: 1px solid white;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 0;
+  margin-left: 20px;
+  margin-bottom: 10px;
+  width: 200px;
+  height: 40px;
 }
-.fa-envelope{
-    font-family: Inter;
-    color: white;
-    margin: auto 30px;
 
+.illustration {
+  height: 290px;
+  width: 375px;
+  background-image: url("/portfolio-first-project/images/Illustration.png");
```

Comment:
The is not working because you are using the wrong path `/portfolio-first-project/images/Illustration.png"`
The correct path is `./images/Illustration.png"`. Kindly fix the relative path.